Vision/README.md

```diff
@@ -1,6 +1,4 @@
-A New Focus for the W3C: Improving the Web’s Integrity
```

Comment:
I dropped this line entirely.  The integrity point is made below, in "Mission", via "we must become more principled in our execution of the vision."

Reply:
Not sure I agree, but don't want to bikeshed about "integrity".  My hope as an early author/editor of this document was to motivate W3C to "Pivot to Integrity". Probably not a good phrase, but the idea was to move W3C's vision from maximizing the POTENTIAL of the future web were were envisioning , and to the STEWARDSHIP the actual web we had built.

Reply:
I think that pivot is represented in this edit, but I'd welcome how to underscore it more.

Reply:
Ahh, I see you took out the "pillars" and replaced with "We will improve the fundamental integrity of the Web platform. The Web will not only grow in scope and importance in our lives; it will grow in respect for its users, grow in the trust of its users, grow in its inclusion of all humanity as its users."

That's fine

Reply:
I think it was supposed to be a one-line title or subtitle; I'd rather have a title that says something about what follows. Let's look for a snappy/catchy informative title that keeps the idea of values being pivotal? (Not a strong objection)